integration-tests/client/chainlink.go

```diff
@@ -74,8 +74,8 @@ func (c *Chainlink) Name() string {
 // CreateJobRaw creates a Chainlink job based on the provided spec string
 func (c *Chainlink) CreateJobRaw(spec string) (*Job, *http.Response, error) {
 	job := &Job{}
-	log.Info().Str("Node URL", c.Config.URL).Msg("Creating Job")
-	log.Trace().Str("Node URL", c.Config.URL).Str("Job Body", spec).Msg("Creating Job")
+	log.Info().Str("Node URL", c.Config.URL).Str("Node Chart", c.Config.ChartName).Msg("Creating Job")
```

Comment:
intentional to put the chart name permanently in info log?

Reply:
Ya, I found it could help make debugging job calls a bit easier